src/components/tailwind-indicator.svelte

```diff
@@ -0,0 +1,20 @@
+<script>
+	let showIndicator = false;
+
+	if (process.env.NODE_ENV === 'development') {
```

Comment:
1. Vite recommended using `import.meta.env.DEV` instead of `process.env.NODE_ENV` (funny enough I thought process.env wasn't gonna work but it did) [source](https://vitejs.dev/guide/env-and-mode.html)
2. Since we assigning a boolean to another boolean, we can just use
```typescript
const showIndicator = import.meta.env.DEV
```

Reply:
Actually, I suggest moving the showing logic out of this component. This component should just be the indicator. Let the component that uses it decide to show it or not, so move the `#if` logic to the layout.